src/style.css

```diff
@@ -1,3 +1,84 @@
+html {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
 body {
+  --color: #00acdc;
+
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  box-sizing: border-box;
+}
+
+header {
+  width: 100%;
+  min-width: 1024px;
```

Comment:
Fixed container sizes can cause issues with responsiveness. Ensure that setting a 'min-width' here aligns with the design requirements and consider using responsive units like percentages or viewport widths if appropriate.